PatientRegistrationApp/src/components/GreetingPage.vue

```diff
@@ -113,6 +109,13 @@ export default {
 					message += " Preferred Language"
 					valid = false
 				}
+				if ((this.isSingleRegistration == false) && (this.isHouseholdRegistration == false)) {
```

Comment:
On line 106, it would be better to write "following field(s):" because now there are two things the user should fill/ click on. 
(wasn't sure how to add a comment on that specific line.. my bad)

Reply:
fixed!  Thanks!

Reply:
I'm resolving this convo as being completed